mission_yaml/SEA44_M72.yml

```diff
@@ -0,0 +1,555 @@
+
+# pld data params
+
+  conductivity:
+    source:       LEGATO_CONDUCTIVITY
+    long_name:    water conductivity
+    standard_name: sea_water_electrical_conductivity
+    units:        mS cm-1
+    coordinates:  time depth latitude longitude
+    instrument:    instrument_ctd
+    valid_min:    "0."
+    valid_max:    "85."
+    observation_type: "measured"
+    accuracy:      "0.003"
+    resolution:     "0.001"
+
+  temperature:
+    source:       LEGATO_TEMPERATURE
+    long_name:    water temperature
+    standard_name: sea_water_temperature
+    units:        Celsius
+    coordinates:  time depth latitude longitude
+    instrument:   instrument_ctd
+    valid_min:    "-5"
+    valid_max:    "42"
+    observation_type: "measured"
+    accuracy:      "0.002"
+    resolution:    "0.00005"
+
+  pressure:
+    source:       LEGATO_PRESSURE
+    long_name:    water pressure
+    standard_name:  sea_water_pressure
+    units:        dbar
+    coordinates:  time depth latitude longitude
+    valid_min:    "0"
+    valid_max:    "1000"
+    positive:      "down"
+    reference_datum:  "sea-surface"
+    instrument:     "instrument_ctd"
+    observation_type: "measured"
+    accuracy:         "0.5"
+    resolution:       "0.01"
+    comment:          "ctd pressure sensor"
+
+  salinity:
+    source:       LEGATO_SALINITY
+    long_name:    water salinity
+    standard_name: sea_water_salinity
+    units:        PSU
+    coordinates:  time depth latitude longitude
+    instrument:    instrument_ctd
+    valid_min:    "0."
+    valid_max:    "50."
+    observation_type: "calculated"
+    accuracy:      "0.02"
+    resolution:     "0.01"
+
+# optics:
+  chlorophyll:
+    source:       FLBBPC_CHL_SCALED
+    long_name:    chlorophyll
+    standard_name: concentration_of_chlorophyll_in_sea_water
+    units:        mg m-3
+    coordinates:  time depth latitude longitude
+    instrument:    instrument_flourometer
+    valid_min:    "0."
+    valid_max:    "50."
+    observation_type: "calculated"
+    sensitivity:      "0.025"
+
+  chlorophyll_raw:
+    source:       FLBBPC_CHL_COUNT
+    long_name:    raw chlorophyll
+    standard_name: raw_chlorophyll_in_sea_water
+    units:        counts
+    coordinates:  time depth latitude longitude
+    instrument:    instrument_flourometer
+
+  phycocyanin:
+    source:       FLBBPC_PC_SCALED
+    long_name:    phycocyanin
+    standard_name: concentration_of_phycocyanin_in_sea_water
+    units:        mg m-3
+    coordinates:  time depth latitude longitude
+    instrument:    instrument_flourometer
+    valid_min:    "0."
+    valid_max:    "175."
+    observation_type: "calculated"
+    sensitivity:      "0.09"
+
+  phycocyanin_raw:
+    source:       FLBBPC_PC_COUNT
+    long_name:    raw phycocyanin
+    standard_name: raw_phycocyanin_in_sea_water
+    units:        counts
+    coordinates:  time depth latitude longitude
+    instrument:    instrument_flourometer
+
+  backscatter:
```

Comment:
One small change. I've changed the name of this backscatter variable to `backscatter_scaled` in prep for proper backscatter processing. This block is now:

```
  backscatter_raw:
    source:       FLBBPC_BB_700_COUNT
    long_name:    raw 700 nm wavelength backscatter
    standard_name: raw_700_nm_wavelength_backscatter_in_sea_water
    units:        counts
    coordinates:  time depth latitude longitude
    instrument:    instrument_scatterometer
```